build/cekit/image.yaml

```diff
@@ -0,0 +1,50 @@
+- name: "builder-infinispan-operator"
+  description: "Builder for Infinispan Operator"
+  version: latest
+  from: "fedora:29"
+  labels:
+    - name: "maintainer"
+      value: "jboss-support-dg6@redhat.com"
```

Comment:
Most of this values are product specific. I'd leave those out for the community image and add them in the product overrides, is that possible?

Reply:
yes sure, done